osgp/platform/osgp-adapter-ws-smartmetering/src/main/resources/db/migration/V20210702135657832__Adds_table_application_key_configuration.sql

```diff
@@ -0,0 +1,21 @@
+DO
+$$
+BEGIN
+
+IF NOT EXISTS (
+    SELECT 1
+    FROM   pg_tables
+    WHERE  schemaname = current_schema
+    AND    tablename  = 'application_key_configuration') THEN
+
+    CREATE TABLE application_key_configuration (
+        organisation_identification VARCHAR(40) NOT NULL,
+        application_name VARCHAR(40) NOT NULL,
+        public_key_location VARCHAR(2000),
```

Comment:
If I am not mistaken the location columns in table `notification_web_service_configuration` were left nullable because the entire trust store and/or key store "sections" are not required to be present.
For the `application_key_configuration` the `public_key_location` is the only data that could be set, and I think something would be broken if this were allowed to be null.
In the scenario of getting the keys for instance, I don't think not encrypting the keys if there is no location for the public key file is a viable option, so an exception would occur because the system configuration does not allow the correct actions to be taken.

Reply:
Added NOT NULL